typegate/engine/src/snapshot.rs

```diff
@@ -0,0 +1,26 @@
+// Copyright Metatype OÜ, licensed under the Elastic License 2.0.
+// SPDX-License-Identifier: Elastic-2.0
+
+// Copyright Metatype OÜ, licensed under the Mozilla Public License Version 2.0.
+// SPDX-License-Identifier: MPL-2.0
+use deno::deno_runtime::ops::bootstrap::SnapshotOptions;
+use mt_deno::deno;
+
+pub fn create_snapshot(snapshot_path: std::path::PathBuf) {
+    // let cli_snapshot_path = PathBuf::from(env::var_os("OUT_DIR").unwrap()).join("CLI_SNAPSHOT.bin");
+
+    let snapshot_options = SnapshotOptions {
+        // NOTE: keep this in sync
```

Comment:
./dev/lock.ts can do it?

Reply:
🙌🏿 but I have removed all snapshotting code now.